Services/IotHub/IotHubConnectionStringManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Runtime;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.IotHub
+{
+    // retrieves Iot Hub connection secret from storage
+    public class IotHubConnectionStringManager
+    {
+        private const string CONNECTION_STRING_FILE_PATH = @"keys\user_iothub_key.txt";
```

Comment:
the value is duplicated in IotHubConnectionStringManager, we should store similar constants in one place